### DISCUSSION TITLE
How can I use sliding window infer for binary classification task?

### DISCUSSION THREAD

**Discussion by SoyaOda** [Project-MONAI/MONAI#3825] [Q&A]
Could you kindly explain how to use sliding window infer for binary classification task from 3D image?
I checked the spleen tutorial and tried to use "sliding_window_inference" but could not use the function for my binary classification task. 
I think tensor size is different because my model change (N,C,image shape) to (N,C,binary one_hot vector).
Is there other methods to use sliding window infer for binary classification?

**Comment by Nic-Ma** [comment]
Hi @SoyaOda ,

I think we don't support sliding window in classification task so far.
But you are making an interesting point, we can consider this feature.
CC @wyli @ericspod .

Thanks.

**Reply by YazdanSalimi** [reply]
Thank you for your response. Any update?

**Reply by wyli** [reply]
yes more recent versions support this use case, for example https://github.com/Project-MONAI/MONAI/discussions/4318#discussioncomment-4786965
